addons/im_livechat/static/src/js/im_livechat.js

```diff
@@ -304,6 +305,7 @@ var Feedback = Widget.extend({
         this.channel_uuid = channel_uuid;
         this.server_origin = session.origin;
         this.rating = undefined;
+        this.closeChatDropPrevious = new concurrency.DropPrevious();
```

Comment:
rename attribute `this.dp` (conventional name for drop previous in some other places)